.dockerignore

```diff
@@ -1,13 +1,17 @@
 # Ignore everything...
 *
+*/assets/media*
```

Comment:
yes .... because I didn't want it accidentally bundling media I had from testing locally... but this is more of a problem that will be solved when we switch over.